### DISCUSSION TITLE
All lfg bots are geared to the teeth

### DISCUSSION THREAD

**Discussion by boldoran** [mod-playerbots/mod-playerbots#379] [Q&A]
All the bots that join my dungeon trough lfg are geared to the teeth, full rare at lvl 20...
Is there a way to randomize this? Because this makes the dungeons way too easy.

Thanks 😀

**Comment by hermensbas** [comment]
Maybe "AiPlayerbot.AutoGearQualityLimit" might help?

**Reply by boldoran** [reply]
i believe that changes all the gear, it would be nice if it was "randomized" So some green, some rare etcetra

**Comment by Dreathean** [comment]
Liyunfan recently pushed changes that lets you set them to uncommon

**Reply by boldoran** [reply]
that still wouldnt make it random as in, some green, some blue etc
